documentation/functions/domain/DefaultTTL.md

```diff
@@ -17,5 +17,17 @@ D('example.com', REGISTRAR, DnsProvider('R53'),
 );
 ```
 
+Use `{"ns_ttl":"3600"},` for a 1h default TTL with `NS` entries:
+```javascript
+D('example.com', REGISTRAR, DnsProvider('xyz'),
+  DefaultTTL("4h"),
+  {"ns_ttl":"3600"},
+  NAMESERVER('ns1.provider.com.'),
+  NAMESERVER('ns2.provider.com.'),
+  A('@','1.2.3.4'), // uses default
+  A('foo', '2.3.4.5', TTL(600)) // overrides default
+);
```

Comment:
Rather than "uses default" or "overrides default", the comment should say the exact amount. 

```
('example.com', REGISTRAR, DnsProvider('xyz'),
  DefaultTTL("1d"),
  NAMESERVER_TTL("2d"),
  NAMESERVER('ns1.provider.com.'),  // TTL is 2d
  NAMESERVER('ns2.provider.com.'), // TTL is 2d
  A('@','1.2.3.4'), // TTL is 1d (the default)
  A('foo', '2.3.4.5', TTL("3d")) // TTL is 3d (overrides default)
);
```